Makefile

```
@@ -0,0 +1,50 @@
+.PHONY: help
+help:
+	@echo "---------------HELP-----------------"
+	@echo "- make build --> Create docker image with dependencies needed"
+	@echo "- make run --> Execute docker container database from postgres, and api from image created previusly"
```

you are not executing a container database because your docker-compose.yml file only has the time-tracker-ui service.
```suggestion
	@echo "- make run --> Execute timetracker_ui docker container."
```